app/src/main/res/values/colors.xml

```diff
@@ -88,7 +88,7 @@
   <color name="submitButtonInactiveText">#61000000</color>
   <color name="grey">#CCCCCC</color>
   <!-- INPUT LAYOUT EDITTEXT -->
-  <color name="mtrl_textinput_default_box_stroke_color">#666666</color>
+  <color name="mtrl_textinput_default_box_stroke_color" tools:override="true">#333333</color>
```

Comment:
Are you sure about this change? I think it might be affecting the other UI screens as they use `#666666`

Reply:
Will take @mschanteltc confirmation on this, as this is something, which should be consistent at all places in app.